### DISCUSSION TITLE
FIDI - Reporting Nordigen errors

### DISCUSSION THREAD

**Discussion by MacPaille** [firefly-iii/firefly-iii#5952] [General]
Hi @JC5 
I think you have said reporting the errors is currently in planning so here is a nice example I should be alerted in the mail report:

[2022-04-01 19:00:22] production.ERROR: TransferException: Client error: `GET https://ob.nordigen.com/api/v2/accounts/XXXXXXXXXXXXXXXXXX/details/` resulted in a `400 Bad Request` response:
{"summary":"End User Agreement (EUA) XXXXXXXXXXXXXXXXXX has expired","detail":"EUA was valid for 90 da (truncated...)

I have replaced my value by XXXXXXXXXXXXXXXXXX. That means I have to manually start the importer again to generate a new session for 90 days. So it's important to be informed I have to do it :)

**Comment by JC5** [comment]
Thanks, that's something I can pick up. Cheers!

**Comment by JC5** [comment]
New release will add an error to the report mail. :+1:

**Reply by MacPaille** [reply]
Thanks a lot!
